com.unity.render-pipelines.high-definition/Runtime/Debug/InstancedProbeShader.shader

```diff
@@ -0,0 +1,97 @@
+Shader "Hidden/HDRP/InstancedProbeShader"
+{
+    Properties
+    {
+    }
+
+    SubShader
+    {
+        Tags { "RenderType" = "Opaque" }
+        LOD 100
+        Pass
+        {
+            CGPROGRAM
+            #pragma vertex vert
+            #pragma fragment frag
+            #pragma multi_compile_instancing
+            #include "UnityCG.cginc"
```

Comment:
We must convert this shader to new style SRP way, i.e don't use UnityCg.cginc. Can be done in next PR